src/main/java/com/gregtechceu/gtceu/common/data/GTOres.java

```diff
@@ -406,7 +406,7 @@ public class GTOres {
                     .oreBlock(new VeinBlockDefinition(Chalcopyrite, 5))
                     .oreBlock(new VeinBlockDefinition(Zeolite, 2))
                     .oreBlock(new VeinBlockDefinition(Cassiterite, 2))
-                    .rareBlock(new VeinBlockDefinition(Realgar, 1))
+                    .rareBlock(new VeinBlockDefinition(Realgar, 2))
```

Comment:
this change does literally nothing as there's only 1 rare block. increase `rareBlockChance` instead